src/app/SharpRaven/Data/Breadcrumb.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+using System.Collections.Generic;
+
+using Newtonsoft.Json;
+
+using SharpRaven.Serialization;
+
+namespace SharpRaven.Data {
+    /// <summary>
+    /// Breadcrumb trail.
+    /// </summary>
+    public class Breadcrumb {
+        private readonly DateTime timestamp;
+
+        public Breadcrumb(string category) 
+        {
+            Category = category;
```

Comment:
What happens if `category` is `null`? Should we default to some string or throw an exception?